openfecli/commands/gather.py

```diff
@@ -59,10 +204,20 @@ def legacy_get_type(res_fn):
                 type=click.Path(dir_okay=True, file_okay=False,
                                          path_type=pathlib.Path),
                 required=True)
+@click.option(
+    '--report',
+    type=click.Choice(['dg', 'ddg', 'legs'], case_sensitive=False),
+    default="dg", show_default=True,
+    help=(
+        "What data to report. 'dg' gives maximum-likelihood estimate of "
+        "asbolute deltaG,  'ddg' gives delta-delta-G, and 'leg' gives the "
```

Comment:
```suggestion
        "asbolute deltaG,  'ddg' gives delta-delta-G, and 'legs' gives the "
```
ddg_legs? environments? transformations? raw?

@hannahbaumann @RiesBen please make a choice